setup.py

```diff
@@ -22,12 +28,15 @@
     test_suite='nose.collector',
     tests_require=['nose'],
     keywords='energyplus',
-    include_package_data=True,
-    install_requires=['PyPubSub', 'beautifulsoup4'],
+    include_package_data=True,  # use /MANIFEST.in file for declaring package data
+    install_requires=install_requires,
     entry_points={
-        'console_scripts': [
+        'gui_scripts': [
             'energyplus_regression_runner=energyplus_regressions.runner:main_gui',
         ],
+        'console_scripts': [
+            'energyplus_regression_configure=energyplus_regressions.configure:configure_cli',
+        ],
```

Comment:
A new entry point called `energyplus_regression_configure` will be available after Pip Install.  The user should run this configuration tool to generate the desktop icon if desired.